srcpkgs/diffuse/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'diffuse'
+pkgname=diffuse
+version=0.7.5
+revision=1
+build_style=meson
+hostmakedepends="gettext"
+makedepends="python3-cairo-devel python3-gobject-devel"
+short_desc="Graphical tool for merging and comparing text files"
+maintainer="Dakota Richline <drichline@protonmail.com>"
+license="GPL-2.0-only, FSFAP"
+homepage="https://github.com/MightyCreak/diffuse"
+distfiles="https://github.com/MightyCreak/diffuse/archive/refs/tags/v${version}.tar.gz"
+checksum=df34a7b2842956dc397d8e72f4d744dc50d403b515c5611f54bd137f3554eeb3
+
+post_install() {
+	vlicense COPYING
```

Comment:
These aren't needed for GPL.

Reply:
Thanks! Just to clarify: no license information needs to be installed for GPL packages?

Reply:
Correct.